src/components/dashboard/container/DictionariesDisplay.jsx

```diff
@@ -52,11 +58,29 @@ export class DictionaryDisplay extends Component {
 
   gotoDictionary = (url) => {
     this.props.history.push(url);
-  }
-
+  };
+  handleNextPage = (e) => {
+    e.preventDefault();
+    const { id } = e.target;
+    this.setState({ currentPage: Number(id) });
+  };
+  changeLimit = (value) => {
+    this.setState({ limit: value });
+  };
   render() {
+    const { currentPage } = this.state;
     const { dictionaries, isFetching } = this.props;
     const { searchInput } = this.state;
```

Comment:
How about combining these constants i.e.`const { currentPage, searchInput } = this.state;`